### PR TITLE
Updating article

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/6599

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md [hunk 1]
```diff
@@ -13,45 +13,33 @@ ms.reviewer:
...
-### Instance
-```
-Set-CsTenantMigrationConfiguration [-Tenant <Guid>] [-MeetingMigrationEnabled <Boolean>]
- [-ACPMeetingMigrationTriggerEnabled <Boolean>]
- [-MeetingMigrationSourceMeetingTypes <MeetingMigrationSourceMeetingTypes>]
- [-MeetingMigrationTargetMeetingTypes <MeetingMigrationTargetMeetingTypes>] [-Instance <PSObject>] [-Force]
- [-WhatIf] [-Confirm] [<CommonParameters>]
+Set-CsTenantMigrationConfiguration [-WhatIf] [-MeetingMigrationEnabled <Object>] [-Confirm]
+ [[-Identity] <Object>] [-Tenant <Object>] [-Force] [-Instance <Object>] [-AsJob]
 ```
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md:23]:
```suggestion
### Identity (Default)
```
Set-CsTenantMigrationConfiguration [-WhatIf] [-MeetingMigrationEnabled <Boolean>] [-Confirm]
 [[-Identity] <XdsIdentity>] [-Tenant <Guid>] [-Force] [-AsJob]
```

### Instance
```
Set-CsTenantMigrationConfiguration [-WhatIf] [-MeetingMigrationEnabled <Boolean>] [-Confirm]
[[-Instance] <PSObject>] [-Tenant <Guid>] [-Force] [-AsJob]
```
```

**Review by dariomws — CHANGES_REQUESTED** [review]

File: skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md (hunk 1 above)
Comment (dariomws) [reply skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md:23]: I think that Identity and Instance cannot be used together.

Comment (get-itips) [reply skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md:23]: well, it is the output of Platy, but I can put back the original ParamSets..

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md [hunk 2]
```diff
@@ -162,14 +164,17 @@ Accept pipeline input: False
...
 
-### -ACPMeetingMigrationTriggerEnabled
-{{Fill ACPMeetingMigrationTriggerEnabled Description}}
+### -AsJob
+Indicates that this cmdlet runs as a background job.
+
+When you specify the AsJob parameter, the command immediately returns an object that represents the background job. You can continue to work in the session while the job finishes. The job is created on the local computer and the results from the Skype for Business Online session are automatically returned to the local computer. To get the job results, use the Receive-Job cmdlet.
+
+For more information about Windows PowerShell background jobs, see [about_Jobs]( https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_jobs?view=powershell-6) and [about_Remote_Jobs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_remote_jobs?view=powershell-6).
```
Comment (tseward) [inline skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md:172]: Remove `en-us` from URLs.

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**